bucket/mailspring.json

```
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://getmailspring.com/",
+    "description": "Mailspring: The open-source, extensible email app for Mac, Linux, and Windows with open tracking, link click tracking, contacts enrichment data and more.",
+    "license": "GPL-3.0",
+    "version": "1.5.7",
+    "url": "https://github.com/Foundry376/Mailspring/releases/download/1.5.7/Mailspring-1.5.7-full.nupkg#/dl.7z",
+    "hash": "sha1:9BA1BF2CD32AD6D138460C08D5E31A8093A5BC65",
+    "extract_dir": "lib\\\\net45",
+    "bin": "Mailspring.exe",
+    "shortcuts": [
+        [
+            "Mailspring.exe",
+            "Mailspring"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Foundry376/Mailspring"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Foundry376/Mailspring/releases/download/$version/Mailspring-$version-full.nupkg#/dl.7z",
+        "hash": {
+            "url": "https://github.com/Foundry376/Mailspring/releases/download/1.5.7/RELEASES"
```

- Hardcoded version
- No need to use full url

```suggestion
            "url": "$baseurl/RELEASES"
```